js/ui/marker.js

```diff
@@ -13,6 +13,10 @@ var Point = require('point-geometry');
  * @param {HTMLElement=} element DOM element to use as a marker (creates a div element by default)
  * @param {Object=} options
  * @param {PointLike=} options.offset The offset in pixels as a [`PointLike`](#PointLike) object to apply relative to the element's top left corner. Negatives indicate left and up.
+ * @param {boolean=} [options.pitchScale=false] Whether the marker should scale smaller into the background or larger into the foreground when the map is pitched and rotated
+ * @param {boolean=} [options.zoomScale=false] Whether the marker should scale with the map as you zoom in and out
+ * @param {number=} [options.nativeZoom=map.getZoom()] The native zoom level of the marker. If the map is at this zoom the marker will not be scaled. If zoom is greater the marker will be enlarged. If zoom is less the marker will be shrunk.
+ * @param {number=} [options.zoomFactor=1.6] The multiple by which to scale the size of the marker between each zoom level
```

Comment:
So I'm trying to implement this and I think I'm close..  but how to I get the style functions to overflow past the provided values?

For instance, for your example:
```
var f = glfunc.interpolated({
    stops: [[15, 1], [16, 1.6]],
    base: 2
});
```

This works for the range of [15, 16] as expected:
```
f(15); // => 1
f(15.5); // => 1.248528137423857
f(16); // => 1.6
```
But anything that falls outside that range is bounded by the highest and lowest defined stops:
```
f(14); // => 1
f(17); // => 1.6
```

How do I get it to extrapolate out the correct values beyond the first and last stops without having to rewrite the logic in the constructor?